types/mui-datatables/index.d.ts

```diff
@@ -278,91 +521,266 @@ export type MUIDataTableOptions = Partial<{
+     * Enable/disable fined select column.
+     * @defaultValue true
+     * @link https://github.com/gregnb/mui-datatables/blob/master/examples/fixed-header/index.js
+     */
     fixedSelectColumn: boolean;
+    /**
+     * Enable/disable expansion or collapse on certain expandable rows with custom function.
+     * Returns true if not provided.
+     */
     isRowExpandable: (dataIndex: number, expandedRows?: MUIDataTableIsRowCheck) => boolean;
+    /** Enable/disable selection on certain rows with custom function. Returns true if not provided. */
     isRowSelectable: (dataIndex: number, selectedRows?: MUIDataTableIsRowCheck) => boolean;
+    /**
+     * When true, the option adds a dropdown to the table's footer that allows a user to navigate to a specific page.
+     * @defaultValue false
+     * @link https://github.com/gregnb/mui-datatables/blob/master/examples/large-data-set/index.js
+     */
+    jumpToPage: boolean;
+    /** Callback function that triggers when a cell is clicked. */
     onCellClick: (
         colData: any,
         cellMeta: { colIndex: number; rowIndex: number; dataIndex: number; event: React.MouseEvent },
     ) => void;
     onChangePage: (currentPage: number) => void;
+    /** Callback function that triggers when a page has changed. */
     onChangeRowsPerPage: (numberOfRows: number) => void;
+    /** Callback function that triggers when a column has been dragged to a new location. */
+    onColumnOrderChange: (newColumnOrder: number[], columnIndex: number, newPosition: number) => void;
+    /** Callback function that triggers when a column has been sorted. */
     onColumnSortChange: (changedColumn: string, direction: 'asc' | 'desc') => void;
     /** @deprecated use `onViewColumnsChange` instead */
     onColumnViewChange?: (changedColumn: string, action: string) => void;
     /**
      * A callback function that triggers when the user downloads the CSV file.
      * In the callback, you can control what is written to the CSV file.
      * Return false to cancel download of file.
+     * @link https://github.com/gregnb/mui-datatables/blob/master/examples/on-download/index.js
      */
     onDownload: (
         buildHead: (columns: any) => string,
         buildBody: (data: any) => string,
         columns: any,
         data: any,
     ) => string | boolean;
+    /** Callback function that triggers when filters have changed. */
     onFilterChange: (
         changedColumn: string | MUIDataTableColumn | null,
         filterList: MUIDataTableState['filterList'],
         type: FilterType | 'chip' | 'reset',
+        changedColumnIndex: number,
+        displayData: DisplayData,
     ) => void;
     /**
      * Callback function that is triggered when a user clicks the "X" on a filter chip.
-     * {@link https://github.com/gregnb/mui-datatables/blob/master/examples/serverside-filters/index.js Example}
+     * @link https://github.com/gregnb/mui-datatables/blob/master/examples/serverside-filters/index.js
```

Comment:
```suggestion
     * {@link https://github.com/gregnb/mui-datatables/blob/master/examples/serverside-filters/index.js Example}
```

FYI, @link is always inline. For tagged links it would be @see, but that's not what should be used here.

See:
https://jsdoc.app/tags-see.html and https://jsdoc.app/tags-inline-link.html